configs/templates/unified-dev/spack.yaml

```diff
@@ -60,6 +60,8 @@ spack:
     # Skip neptune-python-env with Intel Classic due to problems
     # with new versions of py-numpy, py-scipy, ...
     - neptune-python-env%intel
+    # Skip incompatible CRTM versions with oneapi
+    - crtm@2.4.0.1%oneapi
```

Comment:
Would it make sense to force it to use GCC instead (in common/packages_oneapi.yaml)? At this point I think this would leave out 2.4.0.1 on a fair number of machines.

Reply:
Hm, yeah. Would it work to build these (seemingly) oneAPI-incompatible packages with Intel Classic for systems where it's avaiable? If it's more trouble than it's worth we can leave it as it is and wait for developers to complain.